.changeset/twelve-elephants-pay.md

```diff
@@ -0,0 +1,36 @@
+---
+'@backstage/plugin-techdocs-backend': minor
+'@backstage/plugin-techdocs-node': minor
+'@techdocs/cli': minor
+---
+
+Add support for default mkdocs plugins.
+
+So far only `techdocs-core` was added as default to the list of mkdocs plugins. In case you use a
+custom image for techdocs, you also might want to add custom default plugins for mkdocs. Unless
+`omitTechdocsCorePlugin: true` is set, the `techdocs-core` will still be joined automatically to the
+set of `defaultPlugins`.
```

Comment:
Hey sorry adding more comments again...

The purpose of this file is to let adopters know what has changed. It's great to provide examples and background, but it doesn't belong in the changeset. They should probably be moved somewhere that will not be buried by other releases.

For this file I think it's enough to say something like "Add optional config and cli option for techdocs to specify default mkdocs plugins"

Reply:
Fine for me - I adjusted it.